### PR TITLE
va - added announcements create page and basic test for it

### LINKED ISSUE #34: #1-F2B - Add admin create announcements page with button and basic tests
Add a button to create an announcement on the Announcements for a specific commons page. It redirects admin to a Create Announcements page. Includes tests for create announcements page.

### PR DESCRIPTION
**Pull request by vaaniarora**
## Overview

In this PR, I added a button to create an announcement for a specific commons that appears on the page for announcements for that commons. I also added a basic test to make sure that the create announcement page renders correctly.

## Screenshots (Optional)

image
image

## Feedback Request (Optional)

Please give suggestions on the following:
- Should I display the commons the new announcement will be for?

## Future Possibilities (Optional)

Some possible points could be:
- This is the first step of making the form on the create announcement page.

## Validation (Optional)

1. Visit https://proj-happycows-va.dokku-08.cs.ucsb.edu
2. Go to Admin -> List commons, scroll horizontally and click on announcements for one of the commons.
3. Click on the create announcements button and see the create announcements page and verify that the url contains the commons id of the announcements page visited.

## Tests

- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #34

### CODE REVIEW

**Review by emilythai — DISMISSED** [review]
LGTM!

File: frontend/src/main/pages/AdminCreateAnnouncementsPage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function AdminCreateAnnouncementsPage() {
+    return (
+        <BasicLayout>
+            <div className="pt-2">
+                <h1>Create Announcement</h1>
```
Comment (pconrad) [inline frontend/src/main/pages/AdminCreateAnnouncementsPage.js:8]:
```suggestion
                <h1>Create Announcement</h1>
                <h2>Not implemented yet; coming soon!</h2>
```

**Review by pconrad — CHANGES_REQUESTED** [review]
One small change to request then this is mergeable

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**